drivers/hmis/lib/form_data/tarrant_county/external_forms/prevention_screening.json

```diff
@@ -10,48 +10,72 @@
           "link_id": "first_name",
           "required": true,
           "type": "STRING",
-          "text": "First Name"
+          "text": "First Name",
```

Comment:
I added `custom_field_key`s here because `introspect_custom_data_element_definitions` skips unless it exists https://github.com/greenriver/hmis-warehouse/blob/release-108/drivers/hmis/app/models/hmis/form/definition.rb#L483

Let me know if it would be preferable to modify that logic instead? Or to make these custom_field_keys more descriptive?